recipes/reproc/all/conanfile.py

```diff
@@ -82,22 +82,21 @@ def package(self):
         rm(self, "*.pdb", self.package_folder, recursive=True)
 
     def package_info(self):
-        # The C++ component should have its separate reproc++-config.cmake file,
-        # but it's currently not supported by Conan
-        self.cpp_info.set_property("cmake_file_name", "reproc")
+        cmake_config_name = "reproc++" if self.options.cxx else "reproc"
+        self.cpp_info.set_property("cmake_file_name", cmake_config_name)
 
-        self.cpp_info.components["reproc_c"].set_property("pkg_config_name", "reproc")
-        self.cpp_info.components["reproc_c"].set_property("cmake_target_name", "reproc")
-        self.cpp_info.components["reproc_c"].libs = ["reproc"]
+        self.cpp_info.components["reproc"].set_property("pkg_config_name", "reproc")
```

Comment:
@czoido This makes the `reproc::reproc` Conan component ambiguous. I'm not even sure whether `cpp_info.requires = ["reproc::reproc"]` and `CMakeDeps.set_property("reproc::reproc", ...)` would interpret this as the aggregate component or the individual one.